docs/src/gs/install-csi.md

```diff
@@ -7,21 +7,53 @@ On Oracle Cloud Infrastructure (OCI), there are two types of storage services av
 
 A persistent volume claim (PVC) is a request for storage, which is met by binding the PVC to a persistent volume (PV). A PVC provides an abstraction layer to the underlying storage. CSI drivers for both the Block Volume Service and File Storage Service have been implemented.
 
-## Configure Authentication
+## Configure authentication via Instance Principal (Recommended)
+
+1. Before downloading the YAML files below, set the version you want to install e.g.
+
+   ```shell
+   export CCM_RELEASE_VERSION=0.13.0
+   ```
+
+2. Download the example configuration file:
+
+   ```shell
+   curl -L https://raw.githubusercontent.com/oracle/oci-cloud-controller-manager/master/manifests/provider-config-example.yaml -o cloud-provider-example.yaml
+   ```
+
+3. Update the auth details as below
```

Comment:
Instead of asking the user to modify the yaml and configure instance_principal, let's provide one with instance_principal already set. Up to you if you want to supply this in CCM repo or in this repo.

Reply:
I dont want to add it in capoci, I will se how we can add it in ccm repo